demos/gazoo.py

```diff
@@ -51,6 +51,40 @@
 # Supporting Classes & Methods
 ##############################################################################
 
+def get_maliput_osm_circuit():
```

Comment:
Can't we make a POD with configurations based on the backend? We can absorb all if-elseif-else cases and collapse them into just one after parsing the selected backend?

This way code will have way less branching and will be much simpler to read and maintain in the future.

Reply:
Let me know if now it is a bit cleaner.